src/character.cpp

```diff
@@ -10242,13 +10242,13 @@ int Character::run_cost( int base_cost, bool diag ) const
                     75; // Mycal characters are faster on their home territory, even through things like shrubs
             }
         }
-        if( is_limb_hindered( hp_leg_l ) ) {
```

Comment:
This function only seems to be used here, so it can now be removed.

Reply:
I have removed the obsolete function.